index.rst

```diff
@@ -1,3 +1,399 @@
+The URL conventions involved are defined in DMTN-076, Internet Endpoints for the Science Platform.
+
+As of early 2020, standing instances are maintained at NCSA at `https://lsst-lsp-stable.ncsa.illinois.edu/portal/app`__ and `https://lsst-lsp-int.ncsa.illinois.edu/portal/app`__, with the latter used preferentially for testing of new releases.
+
+Authentication and Authorization Considerations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Kubernetes ingress rules for the Portal Aspect endpoints include authorization-based redirects as documented in DMTN-094.
+The ingress is configured to require the user's authorization to include the ``exec-portal`` capability.
+The Portal Aspect application is therefore not required to enforce any authorization test itself.
+The application does retrieve the user's personal name from the ``Authorization: Bearer`` token in the post-redirect URL, and displays it in the application's header bar.
+Note that the Portal Aspect implementation is required to be aware of the authorization protocols adopted for LSST in order to be able to construct TAP queries against the LSST services with the proper headers: it must pass on the ``Authorization: Bearer`` token as part of any LSST TAP HTTP requests.
+Presently the query results obtained from the LSST asynchronous TAP service are at redirected URLs on external servers and do not require authorization headers.
+Currently there is no adequate IVOA standard for having the application - or not - of these headers be service-metadata-driven, so, in short, there is contingent, LSST-specific knowledge in the Portal application in this area.
+The present scheme is temporary; at least by the time of operations, it will be required that these result URLs are not universally accessible.
+Portal modifications will be necessary to follow that change.
+This presents a management complication and schedule risk for the LSP, because of the very limited IPAC resources available for maintenance; it will be important for there to be substantial advance notice to IPAC of when any change to this protocol is rolled out.
+
+Note that because the ``exec-portal`` capability is not sufficient on its own to allow for TAP queries (where, at least, ``read-tap`` is required), the technical possibility exists for a user with an unusual set of capabilities to be able to access the Portal Aspect but not perform any TAP (including ObsTAP) queries and therefore not be able to see any LSST data.
+This is not likely to be a situation that arises in practice.
+
+Standalone Firefly Server
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to the Portal Aspect application deployed in the LSP instances in Kubernetes, at this time LSST also maintains a standalone Firefly server, for image visualization purposes, on an NCSA virtual machine at `http://lsst-demo.ncsa.illinois.edu/firefly/`__.
```

Comment:
Although http works, I think we want to point people to https.

Reply:
Fixing.